### PR TITLE
Review & Fix

### PR DESCRIPTION
**Pull request by xudongding**

### CODE REVIEW

**Comment by AnnaTomanek** [comment]
Hi @xudongding ,

Thank you for your contribution and sorry for the delay in handling the PR!

I reviewed your changes and noticed the following issue in a few places:

`{pleaseWait}` is a placeholder - this part will be replaced with the `"pleaseWait": "请稍候"` string when building the application UI. It should be left untouched. 

Can you fix the PR? It will be much appreciated!

File: lang/zh-cn.json [hunk 1]
```diff
@@ -75,33 +75,33 @@
 		"dropMenuItem": "将文件复制至此",
 		"errorDialogTitle": "无法复制文件：",
 		"manyFilesDialogTitle": "复制 {count} 个文件到...",
-		"manyFilesWait": "正复制 {count} 个文件，{pleaseWait}",
+		"manyFilesWait": "正在复制 {count} 个文件。",
```
Comment (zaak) [inline lang/zh-cn.json]:
```suggestion
		"manyFilesWait": "正在复制 {count} 个文件，{pleaseWait}",
```

File: lang/zh-cn.json [hunk 2]
```diff
@@ -35,24 +35,24 @@
...
 		"filesRefresh": "正在刷新文件",
 		"filterPlaceholder": "过滤",
-		"gettingFileData": "正在获取文件数据，{pleaseWait}",
-		"overwrite": "自动覆盖重名文件",
+		"gettingFileData": "正在获取文件数据。",
+		"overwrite": "覆盖",
 		"loadingFilesPane": {
 			"title": "正在加载...",
-			"text": "正在加载文件，{pleaseWait}"
+			"text": "文件加载完成。"
```
Comment (zaak) [inline lang/zh-cn.json]:
```suggestion
			"text": "文件加载完成，{pleaseWait}"
```

File: lang/zh-cn.json [hunk 3]
```diff
@@ -35,24 +35,24 @@
...
-		"fileRenameLabel": "请输入新文件名: ",
+		"fileDeleteConfirmation": "您确定要删除文件 \"{name}\" 吗？",
+		"fileRenameExtensionConfirmation": "改变文件扩展名可能导致文件不可用，确定要更改吗？",
+		"fileRenameLabel": "请输入新文件名：",
 		"filesPaneTitle": "文件窗口",
 		"filesRefresh": "正在刷新文件",
 		"filterPlaceholder": "过滤",
-		"gettingFileData": "正在获取文件数据，{pleaseWait}",
-		"overwrite": "自动覆盖重名文件",
+		"gettingFileData": "正在获取文件数据。",
```
Comment (zaak) [inline lang/zh-cn.json]:
```suggestion
		"gettingFileData": "正在获取文件数据，{pleaseWait}",
```

File: lang/zh-cn.json [hunk 4]
```diff
@@ -75,33 +75,33 @@
 		"dropMenuItem": "将文件复制至此",
 		"errorDialogTitle": "无法复制文件：",
 		"manyFilesDialogTitle": "复制 {count} 个文件到...",
-		"manyFilesWait": "正复制 {count} 个文件，{pleaseWait}",
+		"manyFilesWait": "正在复制 {count} 个文件。",
 		"oneFileDialogTitle": "复制文件到...",
-		"oneFileWait": "正在复制文件，{pleaseWait}",
+		"oneFileWait": "正在复制文件。",
```
Comment (zaak) [inline lang/zh-cn.json]:
```suggestion
		"oneFileWait": "正在复制文件，{pleaseWait}",
```

File: lang/zh-cn.json [hunk 5]
```diff
@@ -75,33 +75,33 @@
...
+		"operationSummary": "已复制的文件数：{count}。"
 	},
 	"move": {
 		"dropMenuItem": "将文件移动至此",
 		"errorDialogTitle": "无法移动文件：",
 		"manyFilesDialogTitle": "移动 {count} 个文件到...",
 		"manyFilesWait": "正在移 {count} 个文件，{pleaseWait}",
 		"oneFileDialogTitle": "移动文件到...",
-		"oneFileWait": "正在移动文件，{pleaseWait}",
+		"oneFileWait": "正在移动文件。",
```
Comment (zaak) [inline lang/zh-cn.json]:
```suggestion
		"oneFileWait": "正在移动文件，{pleaseWait}",
```

**Comment by AnnaTomanek** [comment]
Thanks @zaak for providing the missing placeholder :clap: !

@xudongding, thank you again for your contribution! The updated translations will be available in the next CKFinder release.

**Pull request merged**